lib/proxyquire.js

```diff
@@ -140,6 +153,10 @@ Proxyquire.prototype._resolveModule = function (baseModule, pathToResolve) {
       paths: Module.globalPaths
     })
   } catch (err) {
+    if (this._throwOnNotFound) {
+      throw new ProxyquireError(err.message)
```

Comment:
Might be helpful to indicate which module was not found as well as why the error was thrown (b/c all modules must be registered in the proxyquire statement due to the `throwOnNotFound()` feature)